e2e/Poms/login.ts

```diff
@@ -0,0 +1,38 @@
+import { Page } from "playwright";
+import { expect } from "playwright/test";
+import { LOGIN_SELECTORS } from "../Constants/Selectors/loginpageSelector";
+import { LOGIN_TEXTS } from "../Constants/Texts/loginpageTexts";
+
+export default class LoginPage{
+    constructor(public page:Page){
+        this.page=page
+    }
+
+
+      loginAndVerifyUser=async({email,OTP}:{email:string,OTP:string}):Promise<void>=>{
```

Comment:
The naming convention of OTP is wrong. JS variables need to be named with camelCase and not PascalCase or SCREAMING_SNAKE_CASE.